nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/bin/stream.sh

```diff
@@ -6,12 +6,14 @@ $ROOT/lib/slf4j-log4j12-1.7.21.jar:\
 $ROOT/lib/log4j-1.2.17.jar:\
 $ROOT/nxserver/lib/commons-cli-1.4.jar:\
 $ROOT/nxserver/lib/chronicle-queue-4.6.44.jar:\
-$ROOT/nxserver/lib/zkclient-0.10.jar:\
 $ROOT/nxserver/lib/kafka-clients-1.0.0.jar:\
 $ROOT/nxserver/lib/kafka_2.11-1.0.0.jar:\
 $ROOT/nxserver/lib/scala-library-2.11.7.jar:\
 $ROOT/nxserver/lib/zookeeper-3.4.8.jar:\
 $ROOT/nxserver/lib/jgrapht-core-1.0.1.jar:\
+$ROOT/nxserver/lib/jackson-databind-2.7.4.jar:\
+$ROOT/nxserver/lib/jackson-core-2.7.4.jar:\
+$ROOT/nxserver/lib/jackson-annotations-2.7.4.jar:\
 $ROOT/nxserver/lib/nuxeo-stream-9.10-SNAPSHOT.jar
```

Comment:
This is really not robust and maintainable...

Reply:
Yes I know I have created https://jira.nuxeo.com/browse/NXP-24672 for this